pkg/image/ostree_disk.go

```diff
@@ -30,7 +30,7 @@ type OSTreeDiskImage struct {
 
 	SysrootReadOnly bool
 
-	Remote ostree.Remote
+	Remote *ostree.Remote
```

Comment:
You can guess what I'm going to ask :) Why switch to a pointer instead of just looking for an empty `Name`?

Reply:
I was writing up a whole thing about how it lets us have a nil to signal "no remote at all" but we use parts of it in different places (name, url) so might as well have it concrete and rely on string values.

Reply:
Done.